mgmtsystem_review/models/mgmtsystem_review.py

```diff
@@ -6,6 +6,7 @@
 
 class MgmtsystemReview(models.Model):
     _name = "mgmtsystem.review"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
```

Comment:
for this change you should add the dependency on `mail` no?

Reply:
Good day @simahawk, thanks for your comment. This module depends on `mgmtsystem_survey`, which depends on `survey`, and in turn, depends on `mail`. So we can keep this as is.

Reply:
I always state such direct dependencies explicitly but that's just my pov ;)